README.md

```diff
@@ -114,12 +114,46 @@ salvator-xs-h3-4x2g --ENABLE_ANDROID yes`.
 
 This will require even more time and space, as Android is quite big.
 
+## Building with prebuilt graphics for DomD+DomU
+
+You have three steps to do:
+
+1. Obtain prebuilt graphics binaries from us. Pay attention that you can't use prebuilt binaries from Renesas because those packages do not support virtualization.
```

Comment:
It will be nice to provide some details. Or contacts. Or both.

You can put this details in a separate section and reference it from this place

Reply:
I have reworded this part in more general words, as was agreed by the off-site discussion.